dashboard/app/controllers/curriculum_proxy_controller.rb

```diff
@@ -9,12 +9,23 @@ class CurriculumProxyController < ApplicationController
 
   EXPIRY_TIME = 30.minutes
 
-  # Proxy from studio.code.org/docs/foo to docs.code.org/foo
+  # Proxy from studio.code.org/docs/ to curriculum.code.org/documentation
+  def get_doc_landing
```

Comment:
I couldn't get the get_doc route to work for just studio.code.org/docs and making this separate worked. However if there is a way to make it all one that would be better

Reply:
one thing I noticed is that curriculum.code.org/documentation redirects to curriculum.code.org/documentation/ (trailing slash), but I am guessing that the proxy url renderer does not follow redirects. So your rails code to rewrite studio.code.org/docs to curriculum.code.org/docs is working, but the response is a 301. Does that help?

Reply:
What I would recommend is consolidating the implementations, and adding a comment indicating that URLs like without trailing slashes will not work when running locally. This should be fine because these URLs are only used via iframes within our own product (not typed in by users), and the comment will leave enough of a trail for anyone trying to debug any problems related to this.

Reply:
per offline conversation, let's leave this just how it is and proceed so that we can get our live site off of using docs.code.org. the comments on this method and the next one do a pretty good job of explaining what purpose they each serve.